### ISSUE TITLE
Broken pad due to badChangeset error

### ISSUE THREAD

**Issue by Chocobozzz** [ether/etherpad-lite#3838]
Hello,

We have several broken pads since our 1.8 update and I don't understand why. 
The error on server side is:

```
[2020-04-06 11:19:04.055] [WARN] console - Error: Can't apply USER_CHANGES, because Trying to submit changes as another author in changeset Z:rop>0|d=1ei=2m*0=2$
    at handleUserChanges (/var/www/hebdo.framapad/src/node/handler/PadMessageHandler.js:641:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

The reason is that when we load the pad (without any user interaction), a message is sent on the server:

```
42
[
  "message",
  {
    "type": "COLLABROOM",
    "component": "pad",
    "data": {
      "type": "USER_CHANGES",
      "baseRev": 7723,
      "changeset": "Z:rop>0|d=1ei=2m*0=2$",
      "apool": {
        "numToAttrib": {
          "0": [
            "author",
            "a.dWQUozuAVXf005iq"
          ]
        },
        "nextNum": 1
      }
    }
  }
]
```

I don't understand what this changeset means, and I don't understand why it is sent to the server. However, I know I'm not the `a.dWQUozuAVXf005iq` author.

I don't think it's the same bug than https://github.com/ether/etherpad-lite/issues/2802

Then, we are disconnected and we can't do anything.

~~Broken pad example: https://hebdo.framapad.org/p/framaconfinement-leplan~~

Thank you

**Comment by Chocobozzz** [comment]
Disabling [spellcheck](https://github.com/ether/ep_spellcheck) plugin fixes this issue. 

@JohnMcLear do you want me to close this issue and reopen it on the spellcheck repository?

Steps to reproduce:
 * Install spellcheck plugin
 * Create an empty pad and import 
[framaconfinement-leplan.etherpad.zip](https://github.com/ether/etherpad-lite/files/4442629/framaconfinement-leplan.etherpad.zip)

**Comment by JohnMcLear** [comment]
Thanks @Chocobozzz for this, I will handle, you can leave it with me.

**Comment by JohnMcLear** [comment]
Confirmed, ep_spellcheck at fault.

**Comment by JohnMcLear** [comment]
The problem is due to the latest commit on the spellcheck plugin, it's overly aggressive and modifies the attributes on each span (not each div).  It's caused by a patch applied for https://github.com/ether/ep_spellcheck/issues/11 -- The TLDR here though is browsers are broken when it comes to how they apply spellchecking.  

**spellcheck attribute is not properly passed to children in contentEditable.** cc @rhelmer if you wanna file a browser bug report, you know my stance on spending my time on filing these 👯‍♂

**Comment by JohnMcLear** [comment]
I published a new version of ep_spellcheck which fixes this issue

**Comment by Chocobozzz** [comment]
Thank you @JohnMcLear

**Comment by JocelynDelalande** [comment]
I can confirm bug (on EP 1.7.5) and fix (updating ep_spellcheck to 0.0.7). Thanks @JohnMcLear !

**Comment by rhelmer** [comment]
> The problem is due to the latest commit on the spellcheck plugin, it's overly aggressive and modifies the attributes on each span (not each div). It's caused by a patch applied for [ether/ep_spellcheck#11](https://github.com/ether/ep_spellcheck/issues/11) -- The TLDR here though is browsers are broken when it comes to how they apply spellchecking.
> 
> **spellcheck attribute is not properly passed to children in contentEditable.** cc @rhelmer if you wanna file a browser bug report, you know my stance on spending my time on filing these 👯‍♂

Happy to follow up - @JohnMcLear can you give me more details on how browsers are broken? If we can build a minimal testcase that would be ideal.
